automated install/basic-install.sh

```diff
@@ -1235,6 +1236,11 @@ installConfigs() {
     # Make sure Pi-hole's config files are in place
     version_check_dnsmasq
 
+    # Install an empty regex file
+    touch "${regexFile}"
```

Comment:
Might want to wrap this in a guard just so that it's idempotent for future installs.

```
if [[ ! -f "${regexFile}" ]]; then
  do stuff
fi
```